libs/features/personalization/personalization.js

```diff
@@ -393,6 +397,10 @@ const getVariantInfo = (line, variantNames, variants) => {
         val: normalizePath(line[vn]),
         action,
       });
+    } else if ((action in COMMANDS || action in CREATE_CMDS)
```

Comment:
I wonder if we want to gate it to `action` in COMMANDS / CREATE_CMDS - I could see gnav having its own set of specialized commands that wouldn't need to be defined in personalization.js.

Reply:
It's certainly possible that we could have those in the future. Would it be fine if we defined those somewhere in the global-navigation folder and import them here?